Windows unit tests failing in Bokeh-CI-Full workflow

All Windows unit tests are failing in the `Bokeh-CI-Full` workflow with `test_defaults.py` baseline differences.

**Failing run:** https://github.com/bokeh/bokeh/actions/runs/23608874916

**Failed jobs:**
- unit-test (windows-latest, 3.11)
- unit-test (windows-latest, 3.12)
- unit-test (windows-latest, 3.13)
- unit-test (windows-latest, 3.14)

**Error:** `AssertionError: baseline differs` in `tests/test_defaults.py`

**Root cause:** The workflow configures `git config --global core.autocrlf false` which forces LF line endings on checkout, but Python writes CRLF on Windows, causing git diff to see the entire baseline file as changed.

--- a/.github/workflows/bokeh-ci-full.yml
+++ b/.github/workflows/bokeh-ci-full.yml
@@ -51,6 +51,7 @@ jobs:
         os: [ubuntu-24.04, macos-latest, windows-latest]
 
     steps:
+      # Required for codebase tests that check file formatting (line endings, whitespace, etc).
       - name: Configure line endings in git
         if: runner.os == 'Windows'
         run: git config --global core.autocrlf false
@@ -145,10 +146,6 @@ jobs:
       PYTHON: ${{ matrix.python-version }}
 
     steps:
-      - name: Configure line endings in git
-        if: runner.os == 'Windows'
-        run: git config --global core.autocrlf false
-
       - uses: actions/checkout@v6
 
       - name: Prepare Environment
@@ -179,10 +176,6 @@ jobs:
       OS: ${{ matrix.os }}
 
     steps:
-      - name: Configure line endings in git
-        if: runner.os == 'Windows'
-        run: git config --global core.autocrlf false
-
       - uses: actions/checkout@v6
 
       - name: Prepare Environment
@@ -212,10 +205,6 @@ jobs:
       OS: ${{ matrix.os }}
 
     steps:
-      - name: Configure line endings in git
-        if: runner.os == 'Windows'
-        run: git config --global core.autocrlf false
-
       - uses: actions/checkout@v6
 
       - name: Prepare Environment
@@ -244,10 +233,6 @@ jobs:
         python-version: ['3.11']
 
     steps:
-      - name: Configure line endings in git
-        if: runner.os == 'Windows'
-        run: git config --global core.autocrlf false
-
       - uses: actions/checkout@v6
 
       - name: Prepare Environment

--- a/.github/workflows/bokeh-ci.yml
+++ b/.github/workflows/bokeh-ci.yml
@@ -45,6 +45,7 @@ jobs:
         os: [ubuntu-24.04, macos-latest, windows-latest]
 
     steps:
+      # Required for codebase tests that check file formatting (line endings, whitespace, etc).
       - name: Configure line endings in git
         if: runner.os == 'Windows'
         run: git config --global core.autocrlf false

--- a/.github/workflows/bokeh-ci.yml
+++ b/.github/workflows/bokeh-ci.yml
@@ -45,6 +45,11 @@ jobs:
         os: [ubuntu-24.04, macos-latest, windows-latest]
 
     steps:
+      # Required for codebase tests that check file formatting (line endings, whitespace, etc).
+      - name: Configure line endings in git
+        if: runner.os == 'Windows'
+        run: git config --global core.autocrlf false
+
       - uses: actions/checkout@v6
 
       - name: Prepare Environment

--- a/.github/workflows/bokeh-ci.yml
+++ b/.github/workflows/bokeh-ci.yml
@@ -45,11 +45,6 @@ jobs:
         os: [ubuntu-24.04, macos-latest, windows-latest]
 
     steps:
-      # Required for codebase tests that check file formatting (line endings, whitespace, etc).
-      - name: Configure line endings in git
-        if: runner.os == 'Windows'
-        run: git config --global core.autocrlf false
-
       - uses: actions/checkout@v6
 
       - name: Prepare Environment

